HCP Terraform Docs (#2228)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/workspaces/policy-enforcement/index.mdx

The following workflow describes how to create and manage policies manually. 

### Create a policy set

> Add a policy configuration file to a repository in your version control system (VCS), add policies, then connect them to your organization. The policy configuration file format depends on your policy framework. Refer to the following topics for more information:

Reviewer: ```suggestion
Add a policy configuration file to a repository in your Version Control System (VCS), add policies, then connect them to your organization. The policy configuration file format depends on your policy framework. Refer to the following topics for more information:
```

- [Configure a Sentinel policy set with a VCS repository](/terraform/cloud-docs/workspaces/policy-enforcement/manage-policy-sets/sentinel-vcs)
- [Configure an OPA policy set with a VCS repository](/terraform/cloud-docs/workspaces/policy-enforcement/manage-policy-sets/opa-vcs)

> You can manually write custom policies in Sentinel or OPA framework format or implement pre-written Sentinel policies created and maintained by HashiCorp. Pre-written Sentinal policies enforce common standards, such as PCI DSS. Refer to [Pre-written policy library](/terraform/cloud-docs/workspaces/policy-enforcement/prewritten-library) for information about publicly available pre-written policies. For information about writing custom Sentinel policies, refer to the [Sentinel documentation](/sentinel).

Reviewer: ```suggestion
You can manually write custom policies in Sentinel or OPA framework format, or implement pre-written Sentinel policies created and maintained by HashiCorp. Pre-written Sentinal policies enforce common standards, such as PCI DSS. Refer to [Pre-written policy library](/terraform/cloud-docs/workspaces/policy-enforcement/prewritten-library) for information about publicly available pre-written policies. For information about writing custom Sentinel policies, refer to the [Sentinel documentation](/sentinel).
```

Reviewer: I think we should define `PCI DSS`.

> You can apply policies globally or apply them to specific [projects](/terraform/cloud-docs/projects/manage) and workspaces. For each run in the selected workspaces, HCP Terraform checks the Terraform plan against the policy set. A policy set can only contain policies written in a single policy framework, but you can add Sentinel or OPA policy sets to the same workspace. 

Reviewer: ```suggestion
You can apply policies globally or apply them to specific [projects](/terraform/cloud-docs/projects/manage) and workspaces. For each run in the selected workspaces, HCP Terraform checks the Terraform plan against the policy set. A policy set can only contain policies written in a single policy framework, but you can add Sentinel or OPA policy sets to the same workspace.
```

> Although the UI allows you author and store policies in the application, we recommend storing policies in VCS to implement a policy-as-code workflow, which ensure standardization, security, and auditibility. You can also create policy sets programmatically using the API. Refer to [Managing Policy Sets](/terraform/cloud-docs/policy-enforcement/manage-policy-sets) for details.

Reviewer: ```suggestion
Although the UI allows you author and store policies in the application, we recommend storing policies in a VCS to implement a policy-as-code workflow, which ensures standardization, security, and auditability. You can also create policy sets programmatically using the API. Refer to [Managing Policy Sets](/terraform/cloud-docs/policy-enforcement/manage-policy-sets) for details.
```
